### DISCUSSION TITLE
Capture the flag

### DISCUSSION THREAD

**Discussion by Tumtumz** [FakeFishGames/Barotrauma#6057] [Ideas and suggestions]
Some friends and I were just playing PVP, we had the idea of CTF as a game mode, it would make much more sense than the current "seek and destroy" mission. It would basically be the wreck salvage mission but with 2 competing submarines. That way the 2 subs would have a place to converge on and also have something to chase after, you're never in a situation where you're lost or don't know what to do.

**Comment by Cellachan** [comment]
This is the best idea and I am the best at creating ideas, this rivals one of my own!!
